app/build.gradle

```diff
@@ -0,0 +1,74 @@
+apply plugin: 'com.android.application'
+apply plugin: 'com.github.kt3k.coveralls'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion '22.0.1'
+
+    defaultConfig {
+        applicationId 'burrows.apps.mathapp'
+        minSdkVersion 9
+        targetSdkVersion 22
+        versionCode 1
+        versionName '1.0'
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_7
+        targetCompatibility JavaVersion.VERSION_1_7
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+}
+
+dependencies {
+    compile project(':json')
+    compile 'com.android.support:appcompat-v7:22.1.1'
+    compile 'com.android.support:cardview-v7:22.1.1'
+    compile 'com.android.support:recyclerview-v7:22.1.1'
+
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.hamcrest:hamcrest-integration:1.3'
+    testCompile 'org.apache.maven:maven-ant-tasks:2.1.3' // fixes mac crash
+    testCompile('org.robolectric:shadows-support-v4:3.0-rc2') {
+        exclude group: 'commons-logging', module: 'commons-logging'
+        exclude group: 'org.apache.httpcomponents', module: 'httpclient'
+    }
+}
+
+task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebug']) {
```

Comment:
Thanks to @nenick, he always does a good job keep up to date with all the latest Android plugins and helpful tools.